pymc3/distributions/continuous.py

```diff
@@ -489,43 +489,190 @@ def logcdf(self, value):
         )
 
 
-def cont_fraction_beta(value, a, b, max_iter=200):
-    '''Evaluates the continued fraction form of the incomplete Beta function.
-    Derived from implementation by Ali Shoaib (https://goo.gl/HxjIJx).
+def incomplete_beta_cfe(a, b, x, small):
+    '''Incomplete beta continued fraction expansions
```

Comment:
Does max iter matter?

Reply:
Maybe it is better to implement cfunc with custom op, scan is slow. Is it possible to have closed form gradient?

Reply:
It only matters when there are too few steps for the continued fraction expansion to be close enough for the given float precision. Allowing for more steps is not useful.

Sorry, I'm not sure I understand what you mean by the gradient here.

In any case, the main issue with the previous (partial) implementation was that the numerical convergence was not very good.
This implementation is much better in that sense while being just as slow.

Of course it would be much faster to not do this symbolically...